### DISCUSSION TITLE
Possibility to select other Filament then Printer brand

### DISCUSSION THREAD

**Discussion by Lazers7** [OrcaSlicer/OrcaSlicer#3024] [Ideas]
Question: Is it possible to make it so that ORCA users can select Filament from a different brand than the brand of the 3D printer selected?

Example: After printer Creality K1 is selected, Creality Filament automatically follows, which can then be selected in the box for different Filaments, but no other Filament brand then Creality can be selected.

But if user also have Prusa Filament that he want to select to use in ORCA for Creality K1, but it is not possible to select Prusa Filament in ORCA that way, the settings are locked to only Creality Filament if Printer's Creality is selected.

Suggestion: If the setting is changed to "advanced" then the restriction should be removed so users can use Prusa Filament not only to Prusa Printer but also for other Printer as Creality, but if the setting is not changed to "advanced" then only the same brand of Filament can be used for the selected 3D printer, this to facilitate for beginners.

Note: Prusa Filament settings are already available for Prusa printers in ORCA slicer, other known Filaments are also available there.

**Comment by LJKraus** [comment]
The filament presets for specific brands seem to have much more Filament start G-code on the Advanced tab, which may be not be appropriate for use on another brand or model of printer.
